src/components/ui4/CardUi4.tsx

```diff
@@ -89,7 +89,7 @@ export const CardUi4 = (props: Props) => {
       try {
         await onClose()
       } catch (err) {
-        showError(err)
+        showDevError(err)
```

Comment:
Since this is already inside `EdgeTouchableOpacity`, just delete the whole try / catch block.